build.sbt

```diff
@@ -7,37 +7,147 @@ ThisBuild / developers := List(
 )
 
 val Scala213 = "2.13.16"
-ThisBuild / crossScalaVersions := Seq(Scala213, "3.3.6")
+ThisBuild / crossScalaVersions := Seq("2.12.20", Scala213, "3.3.6")
 ThisBuild / scalaVersion := Scala213 // the default Scala
+
+ThisBuild / githubWorkflowJavaVersions --= List(JavaSpec.temurin("8"), JavaSpec.temurin("11"))
+ThisBuild / tlJdkRelease := Some(17)
 ThisBuild / startYear := Some(2013)
 
-lazy val root = tlCrossRootProject.aggregate(servlet)
+lazy val root = tlCrossRootProject.aggregate(
+  servlet4,
+  servlet5,
+  servlet6,
+  examplesServlet4,
+  examplesServlet5,
+  examplesServlet6,
+)
 
 val asyncHttpClientVersion = "2.12.3"
+val jettyVersion = "12.0.22"
 val http4sVersion = "0.22.15"
-val jettyVersion = "9.4.46.v20220331"
-val servletVersion = "3.1.0"
+val munitCatsEffectVersion = "1.0.7"
+val disciplineMunitVersion = "1.0.9"
+val servletApi6Version = "6.0.0"
+val servletApi5Version = "5.0.0"
+val servletApi4Version = "4.0.4"
+
+val catsEffectVersion = "2.5.5"
+val scalacheckVersion = "1.15.4"
 
-lazy val servlet = project
-  .in(file("servlet"))
+val scalacheckEffectVersion = "1.0.3"
+
+lazy val servlet4 = project
+  .in(file("servlet4"))
   .settings(
+    name := "http4s-servlet4",
```

Comment:
I'm not sure if you like the name `http4s-servlet4`.
Would you prefer `-ee8` instead of the servlet version? Like `http4s-servlet-ee8` or `http4s-servlet-jee8`?